infrastructure/nomad/playbooks/variables/profiles.yml

```diff
@@ -737,6 +737,21 @@ profiles:
       - *mev_commit_provider_emulator_node1_job
       - *datadog_agent_metrics_collector_job
 
+  mainnet:
+    jobs:
+      - *artifacts_job
+      - *datadog_agent_logs_collector_job
+      - *otel_collector_job
+      - *mev_commit_geth_bootnode1_job
+      - *mev_commit_geth_signer_node1_job
+      - *mev_commit_geth_member_node_job
+      - *contracts_deployer_job
+      - *mev_commit_bridge_job
+      - *mev_commit_dashboard_job
```

Comment:
I think I forgot to add a rule in our nginx for the dashboard. We can call it dashboard.mev-commit.xyz maybe. It uses a different port. If possible add this as well.